org.eclipse.jdt.ls.tests/projects/eclipse/hello/src/java/Foo$Bar.java

```diff
@@ -0,0 +1,5 @@
+package java;
```

Comment:
please avoid the `java` package, It's actually illegal to use (I should never have used that in the 1st place)